### DISCUSSION TITLE
People results are not properly displayed when persona option is selected

### DISCUSSION THREAD

**Discussion by manjunathreddy1** [microsoft-search/pnp-modern-search#3497] [Q&A]
**Version used**
4.10.1

**Describe the bug**
People results are not properly displayed when "Show persona card on hover" is selected as shown. Please help to fix this issue.
In the console following error message is coming up but if I disable "Show persona card on hover" results are displayed properly

**To Reproduce**
Detailed steps to reproduce the behavior:
1. Add  search results webpart to the page and connect it to search box. 
2. Configure this webpart to display results from sharepoint local people results
3. Select people layout
4. Enable the option "Show persona card on hover"
5. Search for people who does not have profile picture and then these results will not be displayed properly without name, job title, email etc

**Expected behavior**
If the profile picture does not exists it should display default profile picture, name, job title, email etc.

**Screenshots**
![image](https://github.com/microsoft-search/pnp-modern-search/assets/265137/655bcec7-9016-460a-8159-b4d2c0dd5811)

**Desktop (please complete the following information):**
 - This issue is occurring in both chrome and edge browser

**Additional context**

**Comment by ivy86** [comment]
Have you set the correct Graph API permissions? See here: https://microsoft-search.github.io/pnp-modern-search/usage/search-results/layouts/people/

**Comment by wobba** [comment]
I recommend using `Show persona card on hover (LPC)`. This way you don't have to think about graph permissions. And if using Graph, you need to check permissions and network calls to figure out what goes wrong.

**Reply by gabbsmo** [reply]
What do you mean by "Show persona card on hover (LPC)"? Is it a setting on the results web part? I see nothing with the LPC suffix. I am using the standard People layout. I also have the graph permissions in

**Reply by kasperbolarsen** [reply]
You are not on 4.10.1 or 4.10.2, right ? ;-)

![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/76d23e03-cc1d-4b46-a557-ea7d568349a2)

**Reply by gabbsmo** [reply]
You are right. We are currently on 4.9.3 in two tenants (test and prod) but only have this issue in prod.

**Reply by kasperbolarsen** [reply]
Same version in two tenants, and only the issue in one....that smells very much like an issue with the granted permissions. In PROD this might be a bit tricky but one solution could be to reset this permissions granted to the PnP Modern Search and make a reinstall of the package after that.

**Reply by gabbsmo** [reply]
Tried removing the permissions mentioned in the docs above and reinstalling the package. I could reapprove all permissions except User.Read.All. Network debugger says status 500 when I try to approve it. But I suppose this is probably not related to the PnP Modern Search package but our tenant.

For now the LPC option work. What does LPC stand for anyway?

**Reply by kasperbolarsen** [reply]
Live Person Card
Yannick wrote about it here : https://blog.yannickreekmans.be/spfx-people-search-web-part-based-on-microsoft-graph/
